REVIEWING.md

```diff
@@ -1,99 +1,21 @@
 # Reviewing guide
 
-- [Reviewing guide](#reviewing-guide)
-  - [The Reviewer Role](#the-reviewer-role)
-  - [Values](#values)
-  - [Process](#process)
-    - [Maintainer edge cases](#maintainer-edge-cases)
-    - [General review guidance](#general-review-guidance)
-  - [Checklist](#checklist)
-  - [Reading List](#reading-list)
+## Values and guidelines
 
-## The Reviewer Role
+All reviewers must abide by the [Code of Conduct](CODE_OF_CONDUCT.md) and are also protected by it.
+A reviewer should not tolerate poor behavior and is encouraged to [report any behavior](CODE_OF_CONDUCT.md#How_to_Report) that violates the Code of Conduct.
```

Comment:
This link might change based on my title suggestion in the other file.

How about extending the link to:
```suggestion
A reviewer should not tolerate poor behavior and is encouraged to [report any behavior that violates](CODE_OF_CONDUCT.md#How_to_Report) the Code of Conduct.
```

Reply:
Good, have updated to match the new anchor 👍🏻